compiler/rustc_builtin_macros/messages.ftl

```diff
@@ -199,6 +199,9 @@ builtin_macros_format_use_positional = consider using a positional formatting ar
 
 builtin_macros_global_asm_clobber_abi = `clobber_abi` cannot be used with `global_asm!`
 
+builtin_macros_global_asm_unsupported_operand = the `{$symbol}` operand cannot be used with `global_asm!`
+    .label = the `{$symbol}` operand is not meaningful for global-scoped inline assembly, remove it
```

Comment:
Based on https://doc.rust-lang.org/reference/inline-assembly.html#syntax, it seems like `in`/`out`/`inout`/etc are "direction specifiers", while "operand" includes the variable. So maybe "specifier" rather than "operand" here?

Reply:
I think the word "operand" is used for the whole thing, so all of `x = inout(reg) x,` is an operand. The word "specifier" does not occur on that page, "spec" only in the grammar. 

The section describing these items is https://doc.rust-lang.org/reference/inline-assembly.html#operand-type. It says that "Several types of operands are supported" and then lists `in`, `out` etc. So I think using the word "operand" is the most helpful. 

The error message points to the specifier specifically because it is easy to parse without emitting any unhelpful errors, but the operand as a whole is unsupported.